pandas/_libs/groupby.pyx

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
don't add extraneous things, ok for now

Reply:
This was a dummy commit to force the CI on a TestClipboard failure.